packages/bun-usockets/src/loop.c

```diff
@@ -342,7 +342,7 @@ void us_internal_dispatch_ready_poll(struct us_poll_t *p, int error, int events)
 
                     if (length > 0) {
                         s = s->context->on_data(s, loop->data.recv_buf + LIBUS_RECV_BUFFER_PADDING, length);
-                        
+                        #ifndef WIN32
```

Comment:
why is this disabled on windows? is it because libuv does callback reads? please add a comment so we don't forget.